cpp/Platform.Equality/Platform.Equality.h

```diff
@@ -0,0 +1,10 @@
+#pragma once
+
+#ifndef PLATFORM_EQUALITY
+#define PLATFORM_EQUALITY
+
+#include "Platform.Equality.Any.h"
+#include "Platform.Equality.Base.h"
```

Comment:
Rename this file to match its contents: `Platform.Equality.EqualityComparerFunctionType.h`